python/tests/test_config.py

```diff
@@ -1,20 +1,36 @@
 import unittest
 
 from uagents import Agent
+from uagents.types import AgentEndpoint
+
+MAILBOX_ENDPOINT = "https://agentverse.ai/v1/submit"
+PROXY_ENDPOINT = "https://agentverse.ai/v1/proxy/submit"
+
 
 agents = [
     Agent(),
-    Agent(mailbox="agent_mailbox_key@some_url"),
-    Agent(mailbox={"agent_mailbox_key": "agent_mailbox_key", "base_url": "some_url"}),
-    Agent(mailbox="agent_mailbox_key"),
-    Agent(agentverse="agent_mailbox_key@some_url"),
-    Agent(agentverse="agent_mailbox_key"),
-    Agent(agentverse="http://some_url"),
-    Agent(agentverse="wss://some_url"),
-    Agent(agentverse="ws://some_url"),
-    Agent(agentverse={"agent_mailbox_key": "agent_mailbox_key", "protocol": "wss"}),
+    Agent(mailbox="agent_mailbox_key"),  # type: ignore  (backwards compatibility)
+    Agent(agentverse="http://some_url", endpoint="http://some_url"),
     Agent(agentverse="https://staging.agentverse.ai"),
     Agent(agentverse={"base_url": "staging.agentverse.ai"}),
+    Agent(mailbox=True),
+    Agent(mailbox=True, endpoint="http://some_url"),
+    Agent(proxy=True),
+    Agent(proxy=True, endpoint="http://some_url"),
+    Agent(mailbox=True, proxy=True),
+]
+
+expected_endpoints = [
+    [],
+    [AgentEndpoint(url=MAILBOX_ENDPOINT, weight=1)],
+    [AgentEndpoint(url="http://some_url", weight=1)],
+    [],
+    [],
```

Comment:
Not AgentEndpoint(url="https://staging.agentverse.ai", weight=1)?

Reply:
No, because agentverse configuration is not completely separate from endpoint configuration, which I think is an improvement.

Reply:
I assume you meant "not" = "now

Reply:
Yes, sorry. :smile: